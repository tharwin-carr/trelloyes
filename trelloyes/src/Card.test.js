import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('Card component test', () => {
    //smoke test:
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    //snapshot test:
    it('renders the UI as expected', () => {
        const tree = renderer
        .create(
            <div name='Card'>
                <button type='button'>delete</button>
                <h3>title={'myTitle'}</h3>
                <p>content={'myContent'}</p>
            </div>
        )
        .toJSON();
        expect(tree).toMatchSnapshot();
    });
});