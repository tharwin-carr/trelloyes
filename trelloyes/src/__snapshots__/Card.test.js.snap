// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component test renders the UI as expected 1`] = `
<div
  name="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    title=
    myTitle
  </h3>
  <p>
    content=
    myContent
  </p>
</div>
`;
